confignode/src/main/java/org/apache/iotdb/confignode/persistence/executor/ConfigRequestExecutor.java

```diff
@@ -56,6 +67,8 @@ public class ConfigRequestExecutor {
 
   private final AuthorInfo authorInfo;
 
+  private static final Logger LOGGER = LoggerFactory.getLogger(ConfigRequestExecutor.class);
```

Comment:
generate, Logger add first line in class

Reply:
fixed